<?xml version="1.0" encoding="utf-8"?>

<launch>
  <node name="detect_object_swing" pkg="crane_x7_examples" type="detect_object_swing.py" />
  <node name="bbox_pos_server" pkg="crane_x7_examples" type="bbox_pos_server.py" />
</launch>
